src/views/List.jsx

```diff
@@ -1,16 +1,47 @@
 import { ListItem } from '../components';
+import { useState } from 'react';
 
 /** List component that displays items in a user's shopping cart  */
 export function List({ data }) {
+	const [filteredList, setFilteredList] = useState([]);
```

Comment:
Curious - did you consider other approaches to filtering items other than creating a separate state variable for `filteredList`?

Reply:
Good question!  I'll get back to you as soon as I'm done researching alternative filtering methods ;)

Reply:
@Bustamor definitely not saying you need to change anything! Again, just curious about how you and @sarahmjenkins landed on this solution.

Reply:
I don't totally remember how we landed on this, but I know we started with `filterInput` because whether or not it had a value would indicate whether a filtered list should render, and then I think in trying to figure out how to get the list to render, it felt like something that could pretty easily match the format used for rendering the full list. I'm super interested in other approaches that may be out there!

Reply:
Nice! One idea that comes to mind is to manipulate the original `data` passed in as a prop as opposed to creating another state variable/filtered list. Again, definitely not something that needs to change but just a thought.